geode-core/src/main/java/org/apache/geode/distributed/internal/membership/gms/messenger/JGroupsMessenger.java

```diff
@@ -481,7 +499,9 @@ private void establishLocalAddress() {
         this.jgAddress = new JGAddress(logicalAddress, ipaddr);
       } catch (NoSuchMethodException | InvocationTargetException | IllegalAccessException e) {
         logger
-            .info("Unable to find getPhysicallAddress method in UDP - parsing its address instead");
+            .info("Unable to find getPhysicallAddress method in UDP");
```

Comment:
typo "ll". Since we are now throwing InternalGemFireError should this still be an info log? Any reason to keep a log message here since we are throwing an exception?

Reply:
I'm not sure what typo this is referring to.  The syntax for the catch() requires single bars.

I agree that we no longer need this log statement.